perses/annihilation/relative.py

```diff
@@ -70,6 +70,7 @@ def __init__(self,
                  omitted_terms=None,
                  flatten_torsions=False,
                  rmsd_restraint=False,
+                 impose_virtual_bonds=True,
```

Comment:
This can potentially change the results. I'm confirming with benchmarks results that the possible consequences of this are not anything to be worried about. Will let you know as soon as I have the results.

Reply:
Imposing the virtual bonds by default is important if we ever use RMSD restraints or try to get sensible trajectory snapshots out, so I want to leave this on by default.